### ISSUE TITLE
[BUG] CMC loss in the master

### ISSUE THREAD

**Issue by lzlzlizi** [gingsi/coot-videotext#12]
**Describe the bug**
Hi, it seems in trainer, the CMC loss is not working.
```
# loss += self.compute_cmc_loss(
#     clip_emb_reshape, clip_emb_mask, clip_emb_lens,
#     sent_emb_reshape, sent_emb_mask, sent_emb_lens)
```

**Comment by gingsi** [comment]
Hi,

it's turned off during validation for performance reasons. During training it's active.

If you want to make sure it's working, you could change one of the configurations and set     loss_cycle_cons: 0   near the top of the file, then check how the performance changes.

**Comment by lzlzlizi** [comment]
Oh, that's right. I didn't notice it's in the eval part. Thanks!

**Comment by gingsi** [comment]
Since the latest update CMC will work correctly both on training and validation.
